pkg/i2gw/providers/gce/implementation_specific.go

```diff
@@ -54,5 +56,6 @@ func implementationSpecificHTTPPathTypeMatch(path *gatewayv1.HTTPPathMatch) {
 	currentValue := *path.Value
 	path.Type = &pmPrefix
 	path.Value = common.PtrTo(strings.TrimSuffix(*path.Value, "/*"))
+	notify(notifications.WarningNotification, fmt.Sprintf("After conversion, ImplementationSpecific Path %s/* will additionally map to %s. See README.md for details.", currentValue, *path.Value))
```

Comment:
maybe instead of README.md, provide a link to the Github to the readme, so the users can click on it

Reply:
Yes that makes more sense. Updated both places, PTAL!